basics/next-app-router/tests/example.spec.ts-snapshots/posthog-events.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Snapshots are great for this. We'd need to bump this if debug output changes, but I don't think this is a bad thing.